adafruit_platformdetect/board.py

```diff
@@ -119,6 +119,8 @@ def id(self):
             board_id = self._clockwork_pi_id()
         elif chip_id == chips.RK3308:
             board_id = self._rock_pi_id()
+        elif chip_id == chips.RK3288:
+            board_id = boards.ASUS_TINKER_BOARD
```

Comment:
Did you mean to use `_asus_tinker_board_id` here?

Reply:
i was having trouble with that earlier trying to get it to pass lint but i think it was more _asus_tinker_board_id function returning something else. it's passing now though so will commit again with that update shortly.

Reply:
Ok, sounds good. I just wanted to make sure.

Reply:
Hi, just curious if you were planning on making this change in this pull request?